package.json

```diff
@@ -12,148 +18,48 @@
-    "babel-plugin-transform-runtime": "~6.23.0",
-    "babel-preset-env": "~1.6.1",
-    "babel-runtime": "~6.26.0",
-    "boxen": "~2.0.0",
-    "chalk": "~2.4.1",
-    "chokidar": "~2.0.4",
-    "contains-path": "~1.0.0",
-    "coveralls": "~3.0.1",
-    "css-loader": "~1.0.0",
-    "danger": "~3.9.0",
-    "debug": "~3.1.0",
-    "debug-error-middleware": "~1.3.0",
-    "dedent": "~0.7.0",
-    "devcert": "~1.0.0",
-    "dotenv": "~6.1.0",
-    "enzyme": "~3.7.0",
-    "enzyme-adapter-react-16": "~1.7.0",
-    "eslint": "~5.2.0",
-    "eslint-plugin-babel": "~5.1.0",
+    "babel-eslint": "~10.0.1",
+    "eslint": "~5.12.0",
+    "eslint-plugin-babel": "~5.3.0",
     "eslint-plugin-graphql": "~3.0.1",
     "eslint-plugin-jsx-a11y": "~6.1.2",
-    "eslint-plugin-node": "~7.0.1",
+    "eslint-plugin-node": "~8.0.1",
     "eslint-plugin-package-json": "~0.1.3",
-    "eslint-plugin-react": "~7.11.1",
-    "execa": "~1.0.0",
-    "express": "~4.16.3",
-    "figures": "~2.0.0",
-    "file-loader": "~2.0.0",
-    "graphql": "~0.13.2",
-    "graphql-tag": "~2.10.0",
-    "hogan.js": "~3.0.2",
-    "http-proxy-middleware": "~0.19.0",
-    "husky": "~1.2.0",
-    "identity-obj-proxy": "~3.0.0",
-    "informed": "~1.10.8",
-    "intl": "~1.2.5",
+    "eslint-plugin-react": "~7.12.3",
+    "graphql": "~14.0.2",
     "jest": "~23.6.0",
-    "jest-fetch-mock": "~2.0.1",
-    "jest-junit": "~5.1.0",
-    "jest-junit-reporter": "~1.1.0",
+    "jest-fetch-mock": "~2.1.0",
+    "jest-junit": "~6.0.1",
     "jest-transform-graphql": "~2.1.0",
-    "js-yaml": "~3.12.0",
-    "keypress": "~0.2.1",
-    "lerna": "~3.4.0",
-    "lodash.debounce": "~4.0.8",
-    "memoize-one": "~4.0.3",
-    "memory-fs": "~0.4.1",
-    "mkdirp": "~0.5.1",
-    "morgan": "~1.9.0",
-    "multispinner": "~0.2.1",
-    "node-fetch": "~2.2.1",
-    "portscanner": "~2.2.0",
     "prettier": "1.15.3",
     "prettier-check": "~2.0.0",
-    "prop-types": "~15.6.2",
-    "react": "~16.6.1",
-    "react-apollo": "~2.2.4",
-    "react-dom": "~16.6.1",
-    "react-feather": "~1.1.5",
-    "react-redux": "~5.0.7",
-    "react-router-dom": "~4.4.0-beta.6",
-    "react-test-renderer": "~16.6.1",
-    "redux": "~4.0.1",
-    "redux-actions": "~2.6.4",
-    "redux-thunk": "~2.3.0",
-    "rimraf": "~2.6.2",
-    "storybook-readme": "~3.3.0",
-    "stream-snitch": "~0.0.3",
-    "strip-ansi": "~5.0.0",
-    "style-loader": "~0.23.0",
-    "supertest": "~3.1.0",
-    "tap-diff": "~0.1.1",
-    "tap-xunit": "~2.3.0",
-    "tape": "~4.9.1",
-    "terser": "~3.8.2",
-    "terser-webpack-plugin": "~1.1.0",
-    "wait-for-expect": "^1.1.0",
-    "webpack": "~4.25.1",
-    "webpack-babel-env-deps": "~1.4.3",
-    "webpack-bundle-analyzer": "~3.0.3",
-    "webpack-cli": "~3.1.2",
-    "webpack-dev-server": "^3.1.14",
-    "workbox-webpack-plugin": "~3.6.1",
-    "write-file-webpack-plugin": "~4.2.0",
-    "xml": "~1.0.1"
-  },
-  "dependencies": {}
```

Comment:
did all these dependencies just get removed? 😳

Reply:
No; they were relocated to the individual packages. Each package should list all of its own dependencies correctly. With `yarn workspaces`, even though those packages are hoisted to the root `node_modules`, they don't need to be added to the root `package.json`.